packages/core/src/html/NodeProcessor.ts

```diff
@@ -353,7 +404,9 @@ class NodeProcessor {
         });
         mainHtmlNodes.forEach(d => NodeProcessor._trimNodes(d));
 
-        const footnotesHtml = this.footnoteProcessor.combineFootnotes(node => this.processNode(node));
+        const footnotesHtml = this.footnoteProcessor.combineFootnotes((node: DomElement) => this.processNode(
+          node, new Context(cwf, [], extraVariables, {}),
```

Comment:
Another functional change. But `context` is unused inside `processNode` here.

For standardising the typing of `processNode` here (2nd parameter is nullable). The other option is to make the 2nd parameter of `processNode(..., context?: Context)`, but that requires some hinting to tsc about whether `context` exists. This seemed like the cleaner option.